CHANGELOG.md

```diff
@@ -13,7 +13,7 @@ All notable changes to this project will be documented in this file.
 [2020-09-16] Version 4.6.5
 --------------------------
 **Library - Docs**
-- [PR #477](https://github.com/sendgrid/sendgrid-java/pull/477): Run *.md documents through Grammer.ly. Thanks to [@vinird](https://github.com/vinird)!
+- [PR #477](https://github.com/sendgrid/sendgrid-java/pull/477): Run *.md documents through Grammar.ly. Thanks to [@vinird](https://github.com/vinird)!
```

Comment:
There's some irony here, since this is a `.md` file, and this change would have been flagged had it been run through grammar.ly.

Reply:
lulz